<launch>

  <node pkg="map_compression" type="errorDistribution" name="errorDistributicon" output="screen">

  <param name="loadMapName" value="/home/yh/mapModel/2018/08.02/learn/Learned_kitti10_map_test_0.1.vtk"/>
ã€€ <param name="saveDisName" value="/home/yh/mapModel/2018/08.02/learn/Learned_kitti10_error_0.1.txt"/>

  </node>
</launch>
